core-blocks/columns/index.js

```diff
@@ -86,7 +142,10 @@ export const settings = {
 					</PanelBody>
 				</InspectorControls>
 				<div className={ classes }>
-					<InnerBlocks layouts={ getColumnLayouts( columns ) } />
+					<InnerBlocks
+						template={ getColumnsTemplate( columns ) }
+						templateLock
```

Comment:
We are setting templateLock as boolean while for the templa_lock we use for CPT's we use a string that allows none, insert or all. A boolean does not allow us to differentiate between a lock all and a lock insert and I feel we should be consistent and also offer both options for innerBlock locking.
Another thing I notice is that we still show the remove button on columns even thougth it does not work (because of the lock).

Should we try to merge PR https://github.com/WordPress/gutenberg/pull/6991 before given that it offers the lock for all, insert and none and also addresses some UI problems?